CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [next]
 
+## 0.20.0
```

Comment:
Leave this on the `[next]` entry, which version will be is defined when we release the version.